File: Morning Snack/Javascript/Strong.js
let inputedNumber = 145;
let number = inputedNumber;
let factorial = 1;
let sum = 0;
while(number != 0){
   let digit = number % 10;
    for(let count = 1; count <= digit; count++){
        factorial *= count;
        sum += factorial;
       console.log(factorial);

}   


 number = number/10;
        
factorial=1;
console.log(sum)
}


if(sum == inputedNumber){

console.log("This is a strong number");

}else{

console.log("This is not a strong number");
}

File: Morning Snack/Java/Strong.java
public class Strong{
public static void main(String... args){
int inputedNumber = 145;
int number = inputedNumber;
int factorial = 1;
int sum = 0;
while(number != 0){
    int digit = number % 10;
    for(int count = 1; count <= digit; count++){
        factorial *= count;
        

}   
 number = number/10;

sum += factorial;
factorial=1;
System.out.println(sum);
}
//System.out.println();
if(sum == inputedNumber){

System.out.println("This is a strong number");

}else{

System.out.println("This is not a strong number");
}


}

}
